src/components/PostInfo/PostInfo.spec.jsx

```diff
@@ -104,51 +104,51 @@
-          email: 'Jayne_Kuhic@sydney.com',
-          body: 'est natus enim nihil est dolore omnis voluptatem numquam\net omnis occaecati quod ullam at\nvoluptatem error expedita pariatur\nnihil sint nostrum voluptatem reiciendis et',
-        },
-      ];
-
-      const otherPost = {
-        userId: 1,
-        id: 3,
-        title: 'ea molestias quasi exercitationem repellat qui ipsa sit aut',
-        body: 'et iusto sed quo iure\nvoluptatem occaecati omnis eligendi aut ad\nvoluptatem doloribus vel accusantium quis pariatur\nmolestiae porro eius odio et labore et velit aut',
-        user: user1,
-        comments: post1Comments,
-      };
-
-      mount(<PostInfo post={otherPost} />);
-
-      cy.get('.PostInfo__title').should(
-        'have.text',
-        'ea molestias quasi exercitationem repellat qui ipsa sit aut',
-      );
-      cy.get('.UserInfo').should('contain', 'Leanne Graham');
-      cy.get('.CommentInfo').should('have.length', 2);
-      cy.get('.CommentInfo__name')
-        .eq(0)
-        .should('have.text', 'id labore ex et quam laborum');
-    });
+    // it('should render another post info', () => {
+    //   const user1 = {
+    //     id: 1,
+    //     name: 'Leanne Graham',
+    //     username: 'Bret',
+    //     email: 'Sincere@april.biz',
+    //   };
+
+    //   const post1Comments = [
+    //     {
+    //       postId: 1,
+    //       id: 1,
+    //       name: 'id labore ex et quam laborum',
+    //       email: 'Eliseo@gardner.biz',
+    //       body: 'laudantium enim quasi est quidem magnam voluptate ipsam eos\ntempora quo necessitatibus\ndolor quam autem quasi\nreiciendis et nam sapiente accusantium',
+    //     },
+    //     {
+    //       postId: 1,
+    //       id: 2,
+    //       name: 'quo vero reiciendis velit similique earum',
+    //       email: 'Jayne_Kuhic@sydney.com',
+    //       body: 'est natus enim nihil est dolore omnis voluptatem numquam\net omnis occaecati quod ullam at\nvoluptatem error expedita pariatur\nnihil sint nostrum voluptatem reiciendis et',
+    //     },
+    //   ];
+
+    //   const otherPost = {
+    //     userId: 1,
+    //     id: 3,
+    //     title: 'ea molestias quasi exercitationem repellat qui ipsa sit aut',
+    //     body: 'et iusto sed quo iure\nvoluptatem occaecati omnis eligendi aut ad\nvoluptatem doloribus vel accusantium quis pariatur\nmolestiae porro eius odio et labore et velit aut',
+    //     user: user1,
+    //     comments: post1Comments,
+    //   };
+
+    //   mount(<PostInfo post={otherPost} />);
+
+    //   cy.get('.PostInfo__title').should(
+    //     'have.text',
+    //     'ea molestias quasi exercitationem repellat qui ipsa sit aut',
+    //   );
+    //   cy.get('.UserInfo').should('contain', 'Leanne Graham');
+    //   cy.get('.CommentInfo').should('have.length', 2);
+    //   cy.get('.CommentInfo__name')
+    //     .eq(0)
+    //     .should('have.text', 'id labore ex et quam laborum');
+    // });
```

Comment:
This test is also commented out. If you want to verify the rendering of another post, consider uncommenting it. Ensure all necessary tests are active to cover different scenarios.